cmd/multi-factor-auth/config/api.toml

```diff
@@ -15,11 +15,11 @@ RestApiInterface = ":8080" # The interface `address and port` to which the REST
 
 [APIPackages.guardian]
     Routes = [
-        { Name = "/register", Open = true, Auth = true },
-        { Name = "/sign-message", Open = true, Auth = false },
-        { Name = "/sign-transaction", Open = true, Auth = false },
-        { Name = "/sign-multiple-transactions", Open = true, Auth = false },
-        { Name = "/verify-code", Open = true, Auth = true },
+        { Name = "/register", Open = true, Auth = true , MaxContentLength = 500 },
+        { Name = "/sign-message", Open = true, Auth = false, MaxContentLength = 500 },
+        { Name = "/sign-transaction", Open = true, Auth = false, MaxContentLength = 500 },
+        { Name = "/sign-multiple-transactions", Open = true, Auth = false, MaxContentLength = 500 },
```

Comment:
i think default value should be higher at least for sign-multiple-transactions, we can have an estimation based on the default value for `MaxTransactionsAllowedForSigning` since these 2 options have to be aligned

Reply:
you are right. let me correct that